cfme/tests/automate/custom_button/test_generic_objects.py

```diff
@@ -345,4 +361,39 @@ def test_custom_button_open_url_evm_obj(setup_obj, button_group):
     Bugzilla:
         1550002
     """
-    pass
+    group, obj_type = button_group
+
+    button = group.buttons.create(
+        text=fauxfactory.gen_alphanumeric(start="btn", separator="_"),
+        hover=fauxfactory.gen_alphanumeric(),
+        open_url=True,
+        system="Request",
+        request=method.name,
+    )
+    request.addfinalizer(button.delete_if_exists)
+
+    view = navigate_to(setup_obj, "Details")
+    custom_button_group = Dropdown(view, group.hover)
```

Comment:
@digitronik Why is this `Dropdown` widget not part of `setup_obj`'s view?

Reply:
We already discussed about this @john-dupuy 
https://github.com/ManageIQ/integration_tests/pull/9020#discussion_r299924973

Reply:
Ah, I knew that looked familiar :grin: